templates/pdns-chroot-resolv.conf.j2

```diff
@@ -0,0 +1,4 @@
+[Service]
+PermissionsStartOnly=true
+ExecStartPre=/bin/install -dm755 "{{ pdns_auth_config.chroot }}/etc"
+ExecStartPre=/bin/cp -afL /etc/resolv.conf "{{ pdns_auth_config.chroot }}/etc/resolv.conf"
```

Comment:
Do you mean `ExecStartPost` or is it twice the same option?

Reply:
It is twice the same options